src/Components/FlashSalesCountdownSection/ArrowButtons/ArrowButtons.js

```diff
@@ -0,0 +1,33 @@
+import styles from "./ArrowButtons.module.css";
+const icons = [
+  {
+    arrowClass: "fas fa-arrow-left",
+    containerClass: styles.leftArrow,
+    iconArrowClass: styles.iconLeft,
+    ellipseClass: styles.ellipse15,
+  },
+  {
+    arrowClass: "fas fa-arrow-right",
+    containerClass: styles.rightArrow,
+    iconArrowClass: styles.iconRight,
+    ellipseClass: styles.ellipse16,
+  },
+];
+function ArrowButtons() {
+  return (
+    <div className={styles.ContainerArrow}>
+      {icons.map((icon, index) => (
```

Comment:
index should not be used as a unique key.

Reply:
done .